vcf/vcf.go

```diff
@@ -3,6 +3,8 @@
 // as well as a Vcf struct containing the information from each data line.
 package vcf
 
+const Version = "VCFv4.3"
```

Comment:
Just curious how you picked this version.

Reply:
4.3 is the most recent version (though there is a draft for 4.4)

https://github.com/samtools/hts-specs